src/ucp/wireup/select.c

```diff
@@ -60,15 +61,28 @@ typedef struct {
 typedef struct {
     ucp_wireup_criteria_t criteria;
     uint64_t              local_dev_bitmap;
-    uint64_t              remote_dev_bitmap;
     ucp_md_map_t          md_map;
     unsigned              max_lanes;
 } ucp_wireup_select_bw_info_t;
 
+typedef struct {
+    unsigned         count;
+    ucp_rsc_index_t  dev_index;
+    ucs_sys_device_t sys_dev;
+} ucp_wireup_remote_device_count_t;
```

Comment:
Can we try to think of another naming here?
`count_t` which includes `count` is a bit weird.